### PR TITLE
Increase mocha timeout from the default 2s to 10s. 

### PR DESCRIPTION
**Pull request by lannka**
Also increase browser disconnect tolerance for all testing env to mitigate the flakiness.
Try to address #3521 . The test timed out after 2s.

### CODE REVIEW

File: build-system/config.js [hunk 1]
```diff
@@ -57,38 +57,26 @@ var integrationTestPaths = commonTestPaths.concat([
...
-        timeout: 10000
-      },
-      captureConsole: false
-    }
+var karmaDefault = {
+  configFile: karmaConf,
+  singleRun: true,
+  client: {
+    mocha: {
+      timeout: 10000
```
Comment (cramforce) [inline build-system/config.js]: Would this be what we use in development? Is it a good value there?

Comment (cramforce) [inline build-system/config.js]: I think in development we should use a low value, so slow tests time out and makes them faster.

Comment (lannka) [inline build-system/config.js]: Good point. I set local test back to 2s. PTAL

File: build-system/config.js [hunk 2]
```diff
@@ -57,38 +57,26 @@ var integrationTestPaths = commonTestPaths.concat([
...
-        timeout: 10000
-      },
-      captureConsole: false
-    }
+var karmaDefault = {
+  configFile: karmaConf,
+  singleRun: true,
+  client: {
+    mocha: {
+      timeout: process.env.TRAVIS ? 10000 : 2000
```
Comment (cramforce) [inline build-system/config.js]: Please add a comment here and explain caveats.

**Comment by cramforce** [comment]
LG with a comment. We should add a comment, and observe a bit how it goes.

**Pull request merged**